test/integration/cartridge_nohttp_test.lua

```diff
@@ -0,0 +1,55 @@
+local fio = require('fio')
+local t = require('luatest')
+local g = t.group('cartridge-without-http')
+
+local helpers = require('test.helper')
+
+local function set_export(cluster, export)
+    local server = cluster.main_server
+    return server.net_box:eval([[
+        local cartridge = require('cartridge')
+        local metrics = cartridge.service_get('metrics')
+        local _, err = pcall(
+            metrics.set_export, ...
+        )
+        return err
+    ]], {export})
+end
+
+
+g.test_http_disabled = function()
+    t.skip_if(type(helpers) ~= 'table', 'Skip cartridge test')
+    local cluster = helpers.Cluster:new({
```

Comment:
Cluster initialisation is duplicated across integration tests. We should refactor it and extract it in hepler.